bcgov/biohubbc
# 421 - BCTW Multiple Device Fix

## Links to Jira Tickets

- [SIMSBIOHUB-421](https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-421)

## Description of Changes

- Multiple devices can be created using the same device_id for critters (with different device makes)

## Testing Notes

- Add a critter
- Add a deployment using a device id and a device make
- Add another deployment with same device id and different device make (should succeed)
- Add another deployment with same device id and same device make (should fail)
